### PR TITLE
TokenRequest and TokenRequestProjection are GA now

### PR DESCRIPTION
**Pull request by zshihang**
TokenRequest and TokenRequestProjection are GA now

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:298]: GA features are enabled by default. We can omit the note.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: Please move these feature gates into the next section; they no longer belong under the heading _Feature gates for Alpha or Beta features_.

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks @zshihang 

I've added some advice about making these changes match how SIG Docs normally does them.

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:294]:
```suggestion
To enable and use token request projection, you must specify each of the following
command line arguments to `kube-apiserver`:

```

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:298]: [Suggested text](https://github.com/kubernetes/website/pull/24823/files#r515684800) added.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (zshihang) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: done

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md
Comment (zshihang) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:294]: done

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md
Comment (zshihang) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md]: thanks

**Comment by kikisdeliveryservice** [comment]
As noted on this PR (https://github.com/kubernetes/kubernetes/pull/93258#issuecomment-720622085) the release team has no Enhancement 1205 on record as being GA for this release. Please clarify/file and exception request as necessary.

**Comment by zshihang** [comment]
exception issue is opened https://github.com/kubernetes/enhancements/issues/2126

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:306]: Do NOT remove the history.
Line 155-158 should be moved here.
The goal is to let users see the evolution history of the feature, not just its current state.

**Comment by annajung** [comment]
/hold cancel
exception approved, k/k pr merged

**Comment by liggitt** [comment]
/hold

would like to keep this on hold until KEP updates are made

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (zshihang) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:305]: done. the history was not removed but kept in the last section.

**Comment by zshihang** [comment]
KEP updates merged. https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md

/hold cancel

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**
